sublime-phpunit.py

```diff
@@ -39,13 +39,20 @@ def find_phpunit_config(self, file_name):
         return phpunit_config_path
 
     def run_in_terminal(self, command):
+        settings = sublime.load_settings("Preferences.sublime-settings")
+        terminal_setting = settings.get('phpunit-sublime-terminal', 'Terminal')
+
         osascript_command = 'osascript '
-        osascript_command += '"' + os.path.dirname(os.path.realpath(__file__)) + '/run_command.applescript"'
-        print(osascript_command)
-        osascript_command += ' "' + command + '"'
-        osascript_command += ' "PHPUnit Tests"'
+
+        if terminal_setting == 'Terminal':
+            osascript_command += '"' + os.path.dirname(os.path.realpath(__file__)) + '/run_command.applescript"'
+            osascript_command += ' "' + command + '"'
+            osascript_command += ' "PHPUnit Tests"'
+        else:
```

Comment:
This checks for a setting, and if there isn't one defaults to "Terminal". It will only use iTerm if it's explicitly set in the config